contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1583,31 +1583,6 @@ SELECT 1001 as type,
   NULL::image definition;
 GRANT SELECT ON sys.syscharsets TO PUBLIC;
 
-create or replace view sys.default_constraints
```

Comment:
I deleted this because it seemed like a duplicate since it was defined above the file (seems like a simple mistake, I modified the one defined above this). Let me know if this should be left in.

Reply:
Makes sense! thank you.